props/saber_fett263_buttons.h

```diff
@@ -64,16 +140,83 @@
 #define MOTION_TIMEOUT 60 * 15 * 1000
 #endif
 
+#ifndef FETT263_SWING_ON_SPEED
+#define FETT263_SWING_ON_SPEED 250
+#endif
+
 #ifndef FETT263_LOCKUP_DELAY
 #define FETT263_LOCKUP_DELAY 200
 #endif
 
+#if defined(FETT263_BATTLE_MODE_ALWAYS_ON) && defined(FETT263_BATTLE_MODE_START_ON)
+#error You cannot define both FETT263_BATTLE_MODE_ALWAYS_ON and FETT263_BATTLE_MODE_START_ON
+#endif
+
+#if defined(FETT263_BATTLE_MODE_ALWAYS_ON) && defined(FETT263_SWING_ON_NO_BM)
+#error You cannot define both FETT263_BATTLE_MODE_ALWAYS_ON and FETT263_SWING_ON_NO_BM
+#endif
+
+#if defined(FETT263_BATTLE_MODE_ALWAYS_ON) && defined(FETT263_TWIST_ON_NO_BM)
+#error You cannot define both FETT263_BATTLE_MODE_ALWAYS_ON and FETT263_TWIST_ON_NO_BM
+#endif
+
+#if defined(FETT263_BATTLE_MODE_ALWAYS_ON) && defined(FETT263_STAB_ON_NO_BM)
+#error You cannot define both FETT263_BATTLE_MODE_ALWAYS_ON and FETT263_STAB_ON_NO_BM
+#endif
+
+#if defined(FETT263_BATTLE_MODE_START_ON) && defined(FETT263_SWING_ON_NO_BM)
+#error You cannot define both FETT263_BATTLE_MODE_START_ON and FETT263_SWING_ON_NO_BM
+#endif
+
+#if defined(FETT263_BATTLE_MODE_START_ON) && defined(FETT263_TWIST_ON_NO_BM)
+#error You cannot define both FETT263_BATTLE_MODE_START_ON and FETT263_TWIST_ON_NO_BM
+#endif
+
+#if defined(FETT263_BATTLE_MODE_START_ON) && defined(FETT263_STAB_ON_NO_BM)
+#error You cannot define both FETT263_BATTLE_MODE_START_ON and FETT263_STAB_ON_NO_BM
+#endif
+
+#if defined(FETT263_SWING_ON) && defined(FETT263_SWING_ON_NO_BM)
+#error You cannot define both FETT263_SWING_ON and FETT263_SWING_ON_NO_BM
+#endif
+
+#if defined(FETT263_SWING_ON) && defined(FETT263_SWING_ON_PREON)
+#error You cannot define both FETT263_SWING_ON and FETT263_SWING_ON_PREON
+#endif
+
+#if defined(FETT263_TWIST_ON) && defined(FETT263_TWIST_ON_NO_BM)
+#error You cannot define both FETT263_TWIST_ON and FETT263_TWIST_ON_NO_BM
+#endif
+
+#if defined(FETT263_TWIST_ON) && defined(FETT263_TWIST_PREON)
+#error You cannot define both FETT263_TWIST_ON and FETT263_TWIST_ON_PREON
+#endif
+
+#if defined(FETT263_STAB_ON) && defined(FETT263_STAB_ON_NO_BM)
+#error You cannot define both FETT263_STAB_ON and FETT263_STAB_ON_NO_BM
+#endif
+
+#if defined(FETT263_STAB_ON) && defined(FETT263_STAB_PREON)
+#error You cannot define both FETT263_STAB_ON and FETT263_STAB_ON_PREON
+#endif
+
 #include "prop_base.h"
 #include "../sound/hybrid_font.h"
 
 #undef PROP_TYPE
 #define PROP_TYPE SaberFett263Buttons
 
+EFFECT(dim);
```

Comment:
Some of these are hard to know what they do.
Please add comments that explain what they are used for.

Reply:
I have a CUSTOM SOUNDS SUPPORTED section in the top comments that had more information but I'll add comments to thesel as well.